notebooks/supply_chain/example_utils.py

```diff
@@ -0,0 +1,98 @@
+from typing import List
+
+import numpy as np
+
+from maro.simulator import Env
+
+
+def get_facility_name2id_mapping(env: Env) -> dict:
+    facility_name2id = {}
+    for facility_id, facility_info in env.summary["node_mapping"]["facilities"].items():
+        facility_name = facility_info["name"]
+        facility_name2id[facility_name] = facility_id
+    return facility_name2id
+
+
+def get_unit_name2id_mapping(env: Env) -> dict:
+    unit_name2id = {}
+    for facility_id, facility_info in env.summary["node_mapping"]["facilities"].items():
+        facility_name = facility_info["name"]
+        for unit_name, unit_info in facility_info['units'].items():
+            if unit_info is None:
+                continue
+            if unit_name == "products":
+                for sku_id, sku_info in unit_info.items():
+                    unit_name2id[f"{facility_name}__{unit_name}{sku_id}"] = sku_info["id"]
+                    for prod_name in ('consumer', 'manufacture', 'seller'):
+                        if sku_info[prod_name] is not None:
+                            unit_name2id[f"{facility_name}__{prod_name}{sku_id}"] = sku_info[prod_name]["id"]
+            else:
+                unit_name2id[f"{facility_name}__{unit_name}"] = unit_info["id"]
+    return unit_name2id
+
+
+def get_storage_status(env: Env, facility_info: dict) -> List:
+    storage_nodes = env.snapshot_list["storage"]
+    storage_features = ("capacity", "remaining_space")
```

Comment:
Personally, I prefer to:
1. use the `("capacity", "remaining_space")` directly as the parameter to access the snapshotlist
2. or separate the 1 access into 2, since we finally return [0] [1] separately.

What's your opinion about this kind of coding style?

Reply:
Option 1 sounds better. I will change that.